lineapy/plugins/pipeline_writers.py

```diff
@@ -39,7 +44,7 @@ def __init__(
     ) -> None:
         self.artifact_collection = artifact_collection
         self.keep_lineapy_save = keep_lineapy_save
-        self.pipeline_name = pipeline_name
+        self.pipeline_name = slugify(pipeline_name)
```

Comment:
how was this working without slugify. dont we use this to create {pipeline_name}_module.py?

Reply:
It had worked so far because our examples never involved pipeline names with spaces, which we cannot guarantee for the user (so this fix).